moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -355,46 +413,53 @@ const ProjectView = () => {
                           horizontal: "center",
                         }}
                       >
-                        <MenuItem onClick={handleMenuClose} disabled={true}>
-                          <ListItemIcon>
-                            <Icon fontSize="small">share</Icon>
-                          </ListItemIcon>
-                          <ListItemText primary="Share" />
-                        </MenuItem>
-                        <MenuItem onClick={handleMenuClose} disabled={true}>
-                          <ListItemIcon>
-                            <Icon fontSize="small">favorite</Icon>
-                          </ListItemIcon>
-                          <ListItemText primary="Add to favorites" />
-                        </MenuItem>
-                        <MenuItem onClick={handleRenameClick}>
-                          <ListItemIcon>
-                            <Icon fontSize="small">create</Icon>
+                        <MenuItem
+                          onClick={handleRenameClick}
+                          className={classes.projectOptionsMenuItem}
+                          selected={false}
+                        >
+                          <ListItemIcon
+                            className={classes.projectOptionsMenuItemIcon}
+                          >
+                            <CreateOutlinedIcon />
                           </ListItemIcon>
                           <ListItemText primary="Rename" />
                         </MenuItem>
                         <MenuItem
```

Comment:
@sergiogcx can you conditionally hide the "Place on hold" item if the project is already on hold?

It's not critical, but i think you have all the state here to easily do this.

Reply:
@sergiogcx @johnclary - so how does one take a project off hold? 🤔

Reply:
I guess setting a phase to "Active" works... I would expect the menu item to change to "Take off hold".